packages/subgraph/tests/addresslist-voting/addresslist-voting.test.ts

```diff
@@ -180,30 +179,31 @@ test('Run Addresslist Voting (handleVoteCast) mappings with mock event', () => {
     START_DATE,
     END_DATE,
     SNAPSHOT_BLOCK,
-    MIN_SUPPORT,
-    MIN_TURNOUT,
+    SUPPORT_THRESHOLD,
+    PARTICIPATION_THRESHOLD,
     VOTING_POWER,
-    '1',
-    '0',
-    '1',
+    '2', // yes
+    '0', // no
+    '0', // abstain
```

Comment:
It needed to change because we use the `worstCaseSupport` quantity now for comparision so having 1 yes vote and 1 abstain vote wasn't enough to be sure that the proposal is executable.